.github/workflows/tests.yml

```diff
@@ -88,6 +88,25 @@ jobs:
         shell: sudo -E -u dio bash {0}
         env:
           DEBIAN_FRONTEND: noninteractive
+      - name: Uninstalling Packages
+        run: |
+          [[ -f .no-pacscript-changes ]] && exit ||:
+          export SUDO_USER=$USER
+          export DEBIAN_FRONTEND=noninteractive
+          export GITHUB_ACTIONS=true
+          for changed_file in ${{ steps.files.outputs.added_modified }}; do
+            if [[ ${changed_file} == *".pacscript" ]]; then
+              pacscript_file=`basename "${changed_file}"`
+              package_name="${pacscript_file/.pacscript/ }"
+              package_dir=`dirname "${changed_file}"`
+              echo "Running pacstall -R for ${package_name}..."
+              cd ${package_dir}
+              pacstall --disable-prompts -R ${pacscript_file}
```

Comment:
Is this the pacscript variable without `.pacscript`?

Reply:
No, I think `package_name` would be the correct variable. You can suggest the correct code.